### DISCUSSION TITLE
CSV export

### DISCUSSION THREAD

**Discussion by sgspecker** [jgraph/drawio#3161] [Ideas]
The CSV import feature looks quite interesting. And I appreciate that you have provided a number of examples. Is it feasible to add a "CSV export" capability to the cloud-based tool? From a learning (and implementation) perspective, it would really be handy to draw a sequence and export it to CSV.

**Comment by ibLANtest** [comment]
I also belive that an export & import functionality for an easy to read and modify clear text format is very useful. I can imagine that the definition of that format can be argued, based on users expectation and editing knowledge.

I found the 'csv format to import' method for a, I just want to straight forward modify in a text editor plain and simple and get a result, very promising. In this way users which are scared away from xml have an approachable way to do "data driven drawings".

**Comment by tolleif** [comment]
It would be very useful if there was a CSV Export function to complement the existing CSV import function. Often  large amounts of data in various databases can be visualized with Drawio. However, the master data is not in Drawio drawings. So exporting symbol instances from Drawio to CSV, and then change around data in i.e. Excel before reimporting to Drawio would be fantastic.

**Comment by tomcollis** [comment]
A format is already in use by LucidChart, inter-operability if the format is 'open' would be great.

**Comment by davidjgraph** [comment]
Marking as duplicate of #3729 as it needs an issue. Apply thumbs up there to vote for it.
